packages/teleport-types/src/uidl.ts

```diff
@@ -189,6 +192,23 @@ export type UIDLStyleDefinitions = Record<string, UIDLStyleValue>
 
 export type UIDLEventDefinitions = Record<string, UIDLEventHandlerStatement[]>
 
+export type UIDLLinkURLDefinition = {
+  type: 'url'
+  options: { url: UIDLAttributeValue; newTab: boolean }
+}
+export type UIDLLinkPageDefinition = { type: 'navlink'; options: { routeName: string } }
+export type UIDLLinkMailDefinition = {
+  type: 'mail'
+  options: { mail: string; subject?: string }
```

Comment:
Hey, can we consider adding support for **body** too on the code-generators side alone. Down the line, we can just add it when needed on the UI.